manifests/config.pp

```diff
@@ -1,7 +1,7 @@
 # == Class icingaweb2::config
 #
-class icingaweb2::config {
-  assert_private()
```

Comment:
The update to move to assert_private is extremely new and not included in even the newest release 4.5.1 (https://github.com/puppetlabs/puppetlabs-stdlib/tree/4.5.1).  You would be required to run off the latest release or specific commit you listed above.  This might limit users from being able to utilize this module, would you be ok with having it set with private() for now and updating once a new version of stdlib is cut ?